examples/platform/silabs/main.cpp

```diff
@@ -16,22 +16,44 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-#include "sl_component_catalog.h"
+
+// SL-TEMP: GN cannot use sl_main until it supports sisdk 2025.6
+// sl_system_init is always used for 917 soc
+#if (SL_MATTER_GN_BUILD == 1 || SLI_SI91X_MCU_INTERFACE)
 #include "sl_system_init.h"
```

Comment:
Not sure if this is a good idea but we could put the in clude with the main function to reduce the scope of the changes.

Reply:
Yeah, I thought about it, but I don't like it. I think it's fine to keep with other includes and search for SL-TEMP and delete those once the sisdk is supported with gn